q2_dada2/_denoise.py

```diff
@@ -44,6 +44,10 @@ def _check_featureless_table(fp):
     'trim_left_r': _WHOLE_NUM,
     'max_ee': _NAT_NUM,
     'trunc_q': _WHOLE_NUM,
+    'chimera_method': _SKIP,
+    # TEST chimera_method FOR VALIDITY HERE?
+    # Value should be 'pooled', 'consensus' or 'none'
```

Comment:
We can skip this if we add a predicate to the semantic type (I'll comment on that further down).